src/lib/resolver.js

```diff
@@ -108,7 +108,8 @@ function interpolate(locals, ctx, lang, args, arr) {
     if (typeof cur === 'string') {
       return [localsSeq, valueSeq + cur];
     } else {
-      const [, value] = subPlaceable(locals, ctx, lang, args, cur.name);
+      const [, value] = subPlaceable(
+        locals, ctx, lang, args, cur.name || cur.v);
```

Comment:
why is that cur.name or cur.v? Is that because Identifier has `v` in old AST and `name` in the new one?